tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/HandlerWrapperSetHandlerIntegration.cs

```diff
@@ -49,6 +52,12 @@ public class HandlerWrapperSetHandlerIntegration
     /// <returns>CallTarget state value</returns>
     internal static CallTargetState OnMethodBegin<TTarget>(TTarget instance, ref Delegate handler)
     {
+        if (IsApiKeyMissing.Value)
```

Comment:
Is this definitely the only place you need it? No other handlers?

Reply:
It seems like it, there's no other method which calls our delegates